gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineVersions.java

```
@@ -84,4 +97,52 @@ private static File rootVersionsPropsFile(Project project) {
         }
         return file;
     }
+
+    public static Set<String> getResolvedArtifacts(Project project) {
+        return project.getRootProject().getAllprojects().stream().flatMap(project1 ->
+            project1.getConfigurations().stream().flatMap(configuration -> {
+                try {
+                    return configuration
+                            .getResolvedConfiguration()
+                            .getResolvedArtifacts().stream()
+                            .map(resolvedArtifact ->
+                                    resolvedArtifact.getModuleVersion().getId().getGroup() + ":"
+                                            + resolvedArtifact.getName());
+                } catch (IllegalStateException e) {
+                    //in every case so far, tnose IleegalStateException are ignorable. It's just the specific
```

tnose -> those

---

I'm a bit wary of swallowing these exceptions, I think instead we should check for isCanBeResolved  and not catch this at all